### ISSUE TITLE
Tests for twisted.protocols should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7725]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7725|
|Type|enhancement|
|Created|2014-11-28 10:52:15Z|
|Branch|https://github.com/twisted/twisted/tree/protocols-testcase-names-7725-2|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7725.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/64d/64d7a02bfbf5b00c8c1c390557a264f0af41b1d6/73d6edb035eda890c2730af76570bedab12b863d.patch) (2603 bytes) - added by eeshangarg on 2014-11-28 10:59:12Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7725 7725
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_protocols_testcase_names_7725_2 branches/protocols-testcase-names-7725-2
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1417171935431828 1417171935431828
changetime__1422307731439451 1422307731439451
version__None None
owner__eeshangarg eeshangarg
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch(bugfix_7725.patch​) that changes all of the tests in twisted.protocols to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks & regards,
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [43746]) Branching to protocols-testcase-names-7725.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @adiroiban|
|-|-|

Many thanks for the work on this... 

I have created a branch but it looks like twistedchecker does not accept this patch.
It is not related to your work so I have asked for help over the main twisted list.

Otherwise this is good for merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban commented|
|-|-|

(In [43768]) Branching to protocols-testcase-names-7725-2.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)|@adiroiban set owner to @eeshangarg|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set status to `closed`|
|-|-|

(In [43771]) Merge protocols-testcase-names-7725-2: Rename test cases from twisted.protocols.

Author: eeshangarg
Reviewers: adiroiban
Fixes: [#7725](https://github.com/twisted/twisted/issues/7725)
